src/components/AddTodoProject/AddTodoProject.js

```diff
@@ -0,0 +1,113 @@
+import StyledForm from '../styled-components/StyledForm';
+import StyledSection from '../styled-components/StyledSection';
+import StyledButton from '../styled-components/StyledButton';
+import StyledLabel from '../styled-components/StyledLabel';
+import StyledInput from '../styled-components/StyledInput';
+import StyledTextarea from '../styled-components/StyledTextarea';
+import StyledArticle from '../styled-components/StyledArticle';
+import StyledDiv from '../styled-components/StyledDiv';
+import MySVG from '../MySVG';
+import { useState } from 'react';
+import useStore from '../../hooks/useStore';
+
+export default function AddTodoProject() {
+	const [todonameValue, setTodonameValue] = useState('');
+	const [todonoteValue, setTodonoteValue] = useState('');
+	const [radio, setRadio] = useState('');
+	const addTodo = useStore(state => state.addTodo);
+
+	return (
+		<StyledForm
+			onSubmit={event => {
+				event.preventDefault();
+				addTodo(todonameValue, todonoteValue, radio);
+				setTodonameValue('');
+				setTodonoteValue('');
+				setRadio('');
+			}}
+		>
+			<StyledSection variant="big-grey">
```

Comment:
It is not semantically correct to put a section / article inside a form. If you want something to separate different parts of the form you should use fieldsets.